source/controller/src/map_ctrl_main.c

```diff
@@ -130,6 +130,10 @@ int main(int argc, char *argv[])
     struct sigaction sig_no_reaction;
     bool             enabled;
 
+#ifdef INCLUDE_BREAKPAD
+    breakpad_ExceptionHandler();
+#endif
+
```

Comment:
Please add Breakpad support, after rdk-easymesh-controller.bb recipe file is merged.